testing/spark/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=spark
+_pkgname=Spark
+pkgver=2.8.3
+pkgrel=0
+pkgdesc="Cross-platform IM client optimized for businesses and organizations."
+url="https://www.igniterealtime.org/projects/spark/"
+arch="all"
+license="Apache-2.0"
+depends="openjdk8-jre-lib portaudio"
+makedepends="apache-ant openjdk8"
+install=""
+subpackages="$pkgname-doc"
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/igniterealtime/$_pkgname/archive/v$pkgver.tar.gz
+	spark.sh
+	"
+options="!check" # No test suite
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	cd "$builddir"/build
+	ant jar
+}
+
+package() {
+	cd "$builddir"
+	install -dm0755 "$pkgdir"/usr/share
+	install -Dm0755 "$srcdir"/spark.sh "$pkgdir"/usr/bin/spark
+	rm -rf target/build/bin
+	rm -rf target/build/logs
+	rm -rf target/build/lib/linux
+	rm -rf target/build/lib/linux64
+	rm -rf target/build/lib/windows
+	rm -rf target/build/lib/windows64
+	cp -R target/build "$pkgdir"/usr/share/$pkgname
+}
+
+doc() {
```

Comment:
Is there a reason you implemented your own doc function? If you need to do something extra, you can call `default_doc` to do the default thing.

Reply:
Thanks for noticing. I removed doc subpackage since help menu uses online documentation.